dagster_scaleway/serverless_job_context.py

```diff
@@ -57,7 +57,7 @@ class ScalewayServerlessJobContext(
         [
             ("docker_image", Optional[str]),
             ("env_vars", Sequence[str]),
-            ("region", str),
+            ("region", Optional[str]),
```

Comment:
note: Region was already optional, this is a miss in the type hints caught by ruff